response_operations_ui/templates/respondent.html

```diff
@@ -40,8 +40,7 @@ <h1 class="jupiter" id="respondent-name">{{ respondent.firstName }} {{ responden
                         <dd id="respondent-account-status">
                             {% if respondent.status  == 'Suspended' %} Locked {% else %} {{ respondent.status }} {% endif %}
                           <br>
-                          <a id="resend-verification-email-btn" href="{{ url_for('respondent_bp.view_resend_verification', respondent_id=respondent.id) }}">Re-send verification email</a>
-                          </br>
+                          {% if respondent.status == 'Suspended' %}<a id="resend-verification-email-btn" href="{{ url_for('respondent_bp.view_resend_verification', respondent_id=respondent.id) }}">Re-send verification email</a> {% endif %}
```

Comment:
- Should show the 'resend email verification' link if the enrolment status is pending i.e.  if a user has just created an account.  
- When the account is locked/suspended, there should be an option to unlock the account rather than resend a verification email.  
- If there's a new (pending) email address then there should be the option to resend the verification email.

Reply:
added this!